Basic Rules.txt

```diff
@@ -665,6 +665,18 @@ Performing a Skill Check (other than Getting Bearings/Awareness of surroundings)
 Conversing With Partners - Free Action
 Getting Your Bearings / Non Skill-Based Perception - Free Action
 
+==== Reactions & Held Actions ====
+Certain abilities or circumstances will allow you to take a special action in response to a 
+triggering event. A reaction occurs at the immediate moment that it is triggered, whether it
+is your turn or not. The most common reaction you will encounter will be from Suppression 
+(described below) or Opportunity Attacks (described above). After the reaction occurs, play 

```

Comment:
"(described above)"
I can't find that description. I see trigger circumstances but no specific definition of exactly what an opportunity attack *is*.
Do you get one attacking action? Can you use any weapon at any range? (probably not heavy weapons at melee range?) Do blindfire rules apply if you don't have your aim set on the offending character? etc.

Reply:
As a note, my requested change may not necessarily seem in-scope from the PR title but I believe that it is in-scope for the referenced issue.